### DISCUSSION TITLE
Customer Portal

### DISCUSSION THREAD

**Discussion by kjuju600** [Peppermint-Lab/peppermint#220] [General]
Thanks for doing this.

Is there a feature for Customer Portal to submit and review their own tickets?

**Comment by lbhopper** [comment]
The dev has an option for customer portal ticket submissions, but I don't see an option for customers/clients to view their submitted tickets. This could be a good feature request...

**Comment by atb007** [comment]
i thought the clients that we are creating with email id could access the portal same as the internal users. Is it not possible?
